python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -185,6 +187,7 @@ def proc_to_examine():
 
     check_process_memory_leaks(proc_to_examine, 25, max_mem_bytes, 80.0)
 
+@SKIP_CONDA_MARK
```

Comment:
What do you think about mentioning why we skip the tests on conda in comment?

E.g. something like:
`# Conda CI runner doesn't have enough storage to perform these stress tests`

This way we'll know we can remove the skip if eventually we increase the capacity on conda.